integration/common

```diff
@@ -147,7 +149,13 @@ deploy_contracts() {
   popd >/dev/null
   title "Migration complete."
 }
-
+deploy_v05_contracts() {
+    log_path=$SRCROOT/integration/initiate-service-agreement.log
+    yarn workspace @chainlink/integration-scripts deploy-v0.5-contracts | tee $log_path
+    export LINK_TOKEN_ADDRESS=`cat $log_path | grep 'Deployed LinkToken at:' | awk '{print$4}'`
```

Comment:
Maybe we should pull the LINK token deployment helper into its own helper? It's duplicated across both deployment helpers, maybe we should just have a third prerequisite helper or check to see if the variable is set or not.

Reply:
This deploys the v0.5 version of the link contract instead, which could be different

Reply:
It shouldn't actually be any different, just duplicated helpers, but both use a helper that has the bytecode hard coded in.

Reply:
So should we by default deploy the v0.5 or v0.4 one?

Reply:
Ideally we'd redeploy after every single test, but that would be a task for later since right now we're reusing everything, including accounts

Reply:
I think the v0.4 one is probably closer to what we want to keep long term, as that package should be all moved over to the new compiler and everything.